cms/db/task.py

```diff
@@ -555,3 +557,8 @@ class Testcase(Base):
     output = Column(
         String,
         nullable=False)
+
+    @validates("codename")
+    def validate_codename(self, key, codename):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/s-maggiolo/cms/file/2228608531/issues/source?bid=3167167&fileBranchId=3175304#l562)